go/inst/instance_dao.go

```diff
@@ -2492,9 +2496,14 @@ var forceFlushInstanceWriteBuffer = make(chan bool)
 
 func enqueueInstanceWrite(instance *Instance, instanceWasActuallyFound bool, lastError error) {
 	if len(instanceWriteBuffer) == config.Config.InstanceWriteBufferSize {
```

Comment:
Should this run if `config.Config.InstanceWriteBufferSize` == 0?

Reply:
oh my bad, added some checks to disable buffer writes if the buffer size is not valid:
https://github.com/github/orchestrator/blob/357fbb86e84bc136f53e41f26fb33e3b3b2fcac9/go/config/config.go#L575-L577